apps/frontend/src/app/5_pages/MarketMakingPage/hooks/useGetTradeVolume.ts

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+
+import { SECONDS_IN_DAY } from '../../../../constants/general';
+import { rskClient } from '../../../../utils/clients';
+import {
+  GetTradeVolumeQuery,
+  useGetTradeVolumeQuery,
+} from '../../../../utils/graphql/rsk/generated';
+
+export const useGetTradeVolume = (pool: string) => {
+  const { loading, data, refetch } = useGetTradeVolumeQuery({
+    client: rskClient,
+    variables: {
+      pool: pool,
+      timestamp: SECONDS_IN_DAY,
```

Comment:
subtract from current day, now it's filtering data since Jan 2, 1970 :)
```suggestion
      timestamp: dayjs().subtract(SECONDS_IN_DAY, 'seconds').unix(),
```